src/day1/greatestarray.c

```diff
@@ -0,0 +1,21 @@
+#include<stdio.h>
+int main()
+{
+      int ar[5],i,j,temp;
```

Comment:
why you always take fix size array? can a user tell the number of elements he wants to enter?
why you haven't initialized the variables, it will take garbage value which may break your program. So always initialize variables